en/includes/guides/authentication/oidc/implement-oidc-hybrid-flow.md

```diff
@@ -0,0 +1,221 @@
+
+- On the application page, go to the **Protocol** tab and take note of the `Client ID` and the `Client secret` generated for the application.
+
+## Enable the hybrid flow
+
+Follow the steps below to enable the hybrid flow for your application.
+
+1. On the {{ product_name }} Console, go to **Applications**.
+2. Select your application and go to its **Protocol** tab.
+3. Under **Allowed grant types**, select **Code**.
+4. Under **Hybrid Flow**, select **Enable Hybrid Flow** and select the preferred response type.
+
+    ![Enable hybrid flow]({{base_path}}/assets/img/guides/authentication/enable-hybrid-flow.png){: width="800" style="display: block; margin: 0; border: 0.3px solid lightgrey;"}
+
+    !!! warning
+
+        It is not recommended to use `code token` and `code id_token token` response types, as obtaining the access token directly from the authorization endpoint without client authentication introduces potential security vulnerabilities, including the risk of account takeover attacks.
+
+5. Click **Update** to save the changes.
+
+## Implement the hybrid flow
+
+Now that you have [enabled the hybrid flow](#enable-the-hybrid-flow) for your application, let's take a look at how you may implement the hybrid flow for your application.
+
+The initial authorization request is similar to that of the [authorization code flow]({{base_path}}/guides/authentication/oidc/implement-auth-code/) with the exception of setting the `response_type` parameter to a value specific to the hybrid flow.
+
+``` bash
+{{host_name}}/oauth2/authorize?
+response_type={response_type}
+&client_id={client_ID}
+&nonce={random_value_generated_by_client}
+&redirect_uri={url_to_redirect_after_login}
+```
+Let's take a closer look at the reponse types available with the OIDC Hybrid Flow.
+
+### code token
+
+Hybrid flow intiated with the `code token` response type requests an authorization code and an access token from the authorization endpoint.
+
+=== "sample request (`code token`)"
+
+    ``` bash
+    {{host_name}}/oauth2/authorize?
+    response_type=code%20token
+    &client_id=SkpwV3lG88X0BU1msAoRRA0zrWEa
+    &nonce=asd
+    &redirect_uri=http://localhost:8080/playground2/oauth2client
+    &scope=openid
+    ```
+=== "sample response (`code token`)"
+
+    ``` bash
+    http://localhost:8080/playground2/oauth2client#
+    access_token=1940a308-d492-3660-a9f8-46723cc582e9
+    &code=99b34587-5483-374d-8b25-50485498e761
+    &token_type=Bearer
+    &expires_in=299999
+    &session_state=baae9a71cdabe38b4643b9d59bd9f65ffaf5a9b8c453f4256c085e5a1c57e624.-EA3ZqPzLvsk25CKmt56YA
+    ```
+
+The token received in the response may be immediately used to invoke APIs authorized for it. The authorization code can be exchanged to receive other tokens such as access tokens, refresh tokens and the ID token as follows.
+
```

Comment:
Before the sample request shall we add the general request format ?

Reply:
<img width="888" alt="image" src="https://github.com/wso2/docs-is/assets/31583807/09e0455d-75dd-417d-87fd-18b1c9e08b52">

Reply:
<img width="933" alt="image" src="https://github.com/wso2/docs-is/assets/31583807/081db2e5-9f65-48e3-9046-2db1fa9c0a4e">